composer.json

```diff
@@ -14,9 +14,9 @@
             "React\\Socket\\": "src"
         }
     },
-    "extra": {
-        "branch-alias": {
```

Comment:
Thanks for spotting, didn't mean to include this in this PR. I've removed these changes and filed https://github.com/reactphp/react/issues/343 instead.